src/webots/nodes/WbViewpoint.cpp

```diff
@@ -1267,27 +1268,27 @@ bool WbViewpoint::moveViewpointToObject(WbBaseNode *node) {
 }
 
 void WbViewpoint::frontView() {
-  orbitTo(WbVector3(0, 0, 1), WbRotation(0, 1, 0, 0));
+  orbitTo(WbVector3(0, -1, 0), WbRotation(0, 0, 1, M_PI_2));
 }
 
 void WbViewpoint::backView() {
-  orbitTo(WbVector3(0, 0, -1), WbRotation(0, 1, 0, -M_PI));
+  orbitTo(WbVector3(0, 1, 0), WbRotation(0, 0, 1, -M_PI_2));
 }
 
 void WbViewpoint::leftView() {
-  orbitTo(WbVector3(-1, 0, 0), WbRotation(0, 1, 0, -M_PI / 2));
+  orbitTo(WbVector3(-1, 0, 0), WbRotation(0, 0, 1, 0));
 }
 
 void WbViewpoint::rightView() {
-  orbitTo(WbVector3(1, 0, 0), WbRotation(0, 1, 0, M_PI / 2));
+  orbitTo(WbVector3(1, 0, 0), WbRotation(0, 0, 1, -M_PI));
 }
 
 void WbViewpoint::topView() {
-  orbitTo(WbVector3(0, 1, 0), WbRotation(1, 0, 0, -M_PI / 2));
+  orbitTo(WbVector3(0, 0, 1), WbRotation(-0.5773, 0.5773, 0.5773, 2.0944));
 }
 
 void WbViewpoint::bottomView() {
-  orbitTo(WbVector3(0, -1, 0), WbRotation(0, -0.707107, 0.707107, M_PI));
+  orbitTo(WbVector3(0, 0, -1), WbRotation(0.5773, -0.5773, 0.5773, 2.0944));
```

Comment:
I think this one is wrong. When comparing it to how it behaves on master (using the `kid.wbt` world). On master, the red team is on the left side, in this PR it's on the right side.

Reply:
Oh, you are right, it should be fixed now